src/gpu-compute/fetch_unit.cc

```diff
@@ -471,6 +494,13 @@ FetchUnit::FetchBufDesc::reserveBuf(Addr vaddr)
 void
 FetchUnit::FetchBufDesc::fetchDone(Addr vaddr)
 {
+    if (vaddr == 0) {
```

Comment:
you should probably add a comment here explaining what vaddr == 0 represents.

A bigger debate is if we can really guarantee vaddr == 0 only for s_icache_inv, but for now probably ok.

Reply:
Added comment